spec/models/admin_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+require 'spec_helper'
+require 'faker'
+
+RSpec.describe Admin, :type => :model do
```

Comment:
wondering if this is needed now that we have a method to create super admins remotely if needed

Reply:
it's true that the fact that admin registration is now accomplished through seeding the database rather than registration on the site is significant, but I think these tests are more important in ensuring the admin schema is sufficient, and I'd therefore argue they're important and should stay